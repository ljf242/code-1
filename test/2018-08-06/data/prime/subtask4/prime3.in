61985928713019 61985938713011 1
