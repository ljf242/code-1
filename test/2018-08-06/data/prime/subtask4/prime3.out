61985928713019
